openssl/src/crypto/rsa.rs

```diff
@@ -162,6 +180,125 @@ impl RSA {
         }
     }
 
+    pub fn max_data(&self) -> Option<u32> {
+        // 41 comes from RSA_public_encrypt(3) for OAEP
+        self.size().map(|len| len - 41)
```

Comment:
This was copied from the old impl, right? There's a (now closed) issue talking about how this is wrong for non-OAEP padding, so we shouldn't have the method unless we can implement it correctly.